main.tf

```diff
@@ -87,5 +91,3 @@ resource "azurerm_log_analytics_solution" "main" {
     product   = "OMSGallery/ContainerInsights"
   }
 }
-
-
```

Comment:
Could we leave these blank lines not changed?

Reply:
Done (my editor automatically remove trailing new lines)